generator/src/lib.rs

```diff
@@ -2032,6 +2047,33 @@ fn derive_setters(
 
                 let mut slice_param_ty_tokens = field.safe_type_tokens(quote!('a), type_lifetime.clone(), None);
 
+                // vkxml wrongly annotates static arrays with a len= field is dynamic. These fields have a static length,
```

Comment:
should be "as dynamic" instead of "is dynamic" ?

Reply:
Neither; I'll fix this up in another PR.